providers/sru/bin/dkms_build_validation.py

```diff
@@ -62,7 +62,11 @@ def parse_dkms_status(dkms_status: str, ubuntu_release: str) -> List[Dict]:
     """
     kernel_info = []
     for line in dkms_status.splitlines():
-        details, status = line.split(": ")
+        details, fullstatus = line.split(": ")
+        if " " in fullstatus:
+            (status, rest) = fullstatus.split(maxsplit=1)
```

Comment:
This `rest` is not propagated anywhere, can we log it somehow?

Reply:
Logged the message